indico/modules/events/papers/models/call_for_papers_test.py

```diff
@@ -0,0 +1,17 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import pytest
+
+from indico.modules.events.papers.settings import paper_reviewing_settings
+
+
+@pytest.mark.usefixtures('dummy_contribution')
+def test_is_authorized_submitter(dummy_user, dummy_event):
+    assert not dummy_event.cfp.is_authorized_submitter(dummy_user)
+    paper_reviewing_settings.set_multi(dummy_event, {'authorized_submitters': {dummy_user}})
```

Comment:
FYI, this should work as well since it's just a single setting:

```python
paper_reviewing_settings.acls.add_principal('authorized_submitters', dummy_user)
```

But I'm fine either way